### ISSUE TITLE
Many elements are not accessible via keyboard

### ISSUE THREAD

**Issue by heull001** [haiwen/seahub#4973]
Many elements like in the actions-column in the library-list are not accessible via keyboard. This is a problem for many disabled people, especially for blind people using screenreaders.

expected: pressing return on such an element should activate it.

Actual: nothing happens.

Please see https://reactjs.org/docs/accessibility.html#mouse-and-pointer-events for further information.

**Comment by freeplant** [comment]
We will work on it soon.
